src/Sylius/Bundle/CoreBundle/Import/Writer/ORM/AbstractDoctrineWriter.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Import\Writer\ORM;
+
+use Doctrine\ORM\EntityManager;
+use Monolog\Logger;
+use Sylius\Component\ImportExport\Model\JobInterface;
+use Sylius\Component\ImportExport\Writer\WriterInterface;
+
+/**
+ * @author Bartosz Siejka <bartosz.siejka@lakion.com>
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+abstract class AbstractDoctrineWriter implements WriterInterface
+{
+    /**
+     * @var array
+     */
+    protected $configuration;
+
+    /**
+     * @var EntityManager
+     */
+    private $entityManager;
+
+    /**
+     * @var Logger
+     */
+    protected $logger;
+
+    /**
+     * @var int
+     */
+    protected $resultCode;
+
+    /**
+     * @param EntityManager $entityManager
+     */
+    public function __construct(EntityManager $entityManager)
+    {
+        $this->entityManager = $entityManager;
+        $this->metadatas['row'] = 0;
+        $this->resultCode = 0;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function write(array $rawUsers, array $configuration, Logger $logger)
```

Comment:
If this is **Abstract**DoctrineWriter, shouldn't array name be more generic? ``$rawItems``?